8_UI/BSML Addons/BSMLUtility.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Xml;
+using System.Linq;
+using System.Collections.Generic;
+using BeatSaberMarkupLanguage;
+using UnityEngine;
+using HarmonyLib;
+
+namespace BeatLeader.UI.BSML_Addons
+{
+    public static class BSMLUtility
```

Comment:
Remove harmony patches and IDs dictionaries. Nerer used but takes a lot of memory and prevents GC of objects\lists